components/centraldashboard/app/metrics.ts

```diff
@@ -0,0 +1,48 @@
+import express, {Request, Response} from 'express';
+import client from "prom-client";
+
+const app = express();
```

Comment:
Is this needed?

Reply:
Hmm I thought that `startMetricsService()` accepted also `app` argument. Would that be possible?

Reply:
I tried that one, however, the `/metrics` API route is already used [here](https://github.com/kubeflow/kubeflow/blob/54201e3d28e4673d465e47e464e86aa698ca0dfa/components/centraldashboard/app/api.ts#L31-L40). We can go with different API routs, e.g. `/prometheus/metrics`. I choose to go with different service, just to make it clear.